### PR TITLE
Added Hide Uploaded Records setting

### PR DESCRIPTION
**Pull request by GaryDaviesGaiaResources**

### CODE REVIEW

File: src/pages/settings/settings.ts [hunk 1]
```diff
@@ -14,6 +15,26 @@ import { IonicPage, NavController, NavParams } from 'ionic-angular';
...
+    public ionViewWillEnter() {
+        this.storageService.getSetting('hideUploaded').subscribe( setting => this.hideUploaded = setting === 'true');
+    }
+
+    public ionViewWillLeave() {
+        this.saveSetting('hideUploaded', this.hideUploaded);
+    }
+
+    private saveSetting(name: string, setting: boolean) {
+        let value = 'false';
```
Comment (tony-gaia) [inline src/pages/settings/settings.ts]: If you need to covert boolean to string, use JSON.stringify

File: src/pages/home/home.ts [hunk 2]
```diff
@@ -100,6 +100,15 @@ export class HomePage {
         while (this.records.length) {
             this.records.pop();
         }
-        this.storageService.getParentRecords().subscribe((record: ClientRecord) => this.records.push(record));
+        this.storageService.getSetting('hideUploaded').subscribe( hideUploaded => {
+            if (hideUploaded === 'true') {
```
Comment (tony-gaia) [inline src/pages/home/home.ts]: If you need to covert string to boolean (or any other valdi JSON), use JSON.parse

Comment (GaryDaviesGaiaResources) [reply src/pages/home/home.ts]: done

File: src/pages/settings/settings.ts (hunk 1 above)
Comment (GaryDaviesGaiaResources) [reply src/pages/settings/settings.ts]: done

**Pull request merged**